comp/otelcol/otlp/map_provider_config_not_serverless.go

```diff
@@ -42,12 +42,17 @@ receivers:
   otlp:
 
 processors:
-  batch:
-    timeout: 10s
   infraattributes:
 
 exporters:
   serializer:
+    sending_queue:
+      sizer: "items"
+      queue_size: 8192
+      batch:
+        flush_timeout: 10000ms
+        min_size: 8192
+        max_size: 0
```

Comment:
Good timing - @liustanley @mx-psi and I are just talking about this in https://dd.slack.com/archives/C086Z7E2A0Y/p1759781049579069. Let's keep the same defaults as other paths: min_size: 10 & max_size: 100